src/Uno.UI.Toolkit/Storage/StorageFile.skia.cs

```diff
@@ -0,0 +1,29 @@
+#nullable enable
+using System.Threading;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using System.Reflection;
+
+namespace Uno.UI.Toolkit;
+
+partial class StorageFileHelper
+{
+	private static Task<bool> FileExistsInPackage(string fileName)
```

Comment:
This code is a duplicate from `StorageFile.Windows.cs`. Consider:

1. Deleting `StorageFile.skia.cs` and `StorageFile.windows.cs`
2. Move the common code to `StorageFileHelper.cs` under `#if __SKIA || WINDOWS || WINDOWS_UWP || WINUI`